README.md

```diff
@@ -1,9 +1,37 @@
+[![Node.js Security WG](https://img.shields.io/badge/Node.js-Security%20WG-green.svg)]()
+[![Security WG Meetings](https://img.shields.io/badge/YouTube-Security%20WG%20Meetings-red.svg)](  https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ/search?query=Security+WG+meeting)
+[![Security WG Twitter Hashtag](https://img.shields.io/badge/Twitter-%23SecurityWG-blue.svg)](https://twitter.com/search?q=SecurityWG)
 [![GitHub Logo](https://img.shields.io/badge/Join%20us%20on-Slack-e01563.svg)](https://nodejs-security-wg.herokuapp.com/)
 [![Security Responsible Disclosure](https://img.shields.io/badge/Security-Responsible%20Disclosure-yellow.svg)](https://github.com/nodejs/security-wg/blob/master/processes/responsible_disclosure_template.md
 )
 
 # Security Working Group
 
+Table Of Contents
+
+- [Charter](#charter)
+- Processes
+  * [Security Announcement Process](./processes/security_annoucement_process.md)
+  * [Security Release Process](./processes/security_release_process.md)
+  * [Node.js CVE management process](./processes/cve_management_process.md)
+  * [Responsible Disclosure Policy](./processes/responsible_disclosure_template.md)
+  * [Third-Party Ecosystem Triage Process](./processes/third_party_vuln_process.md)
+  * [Third-Party HackerOne Submission form](./processes/third_party_vuln_submit_form_hacker1.md)
```

Comment:
I'd probably make this the first item in the list and name it something that will help direct people who are looking to report a vulnerability.

Reply:
+1

Reply:
though that's actually just the template we show on the hackerone landing page... not sure how relevant it will be to put at the top

Reply:
Will go ahead and merge in a few days unless you'd feel the template is really important to place at the top